src/app/MessageDef/CommandPathIB.cpp

```diff
@@ -114,6 +116,24 @@ CHIP_ERROR CommandPathIB::Parser::GetCommandId(chip::CommandId * const apCommand
     return GetUnsignedInteger(to_underlying(Tag::kCommandId), apCommandId);
 }
 
+CHIP_ERROR CommandPathIB::Parser::GetConcreteCommandPath(ConcreteCommandPath & aCommandPath) const
+{
+    ReturnErrorOnFailure(GetGroupCommandPath(&aCommandPath.mClusterId, &aCommandPath.mCommandId));
+
+    return GetEndpointId(&aCommandPath.mEndpointId);
+}
+
+CHIP_ERROR CommandPathIB::Parser::GetGroupCommandPath(ClusterId * apClusterId, CommandId * apCommandId) const
+{
+    ReturnErrorOnFailure(GetClusterId(apClusterId));
```

Comment:
I think we also need to check whether endpointId is there or not in group path, if it is there, it return error.

https://github.com/project-chip/connectedhomeip/issues/29465

Reply:
Let's solve https://github.com/project-chip/connectedhomeip/issues/29465 separately.  In particular, doing that would mean restructuring some other existing code in AttributePathIB, so I would prefer we not mix that in with this PR.

Reply:
sounds good, thanks